apptail/instance.go

```diff
@@ -79,10 +95,16 @@ func (instance *Instance) tailFile(name, filename string, stopCh chan bool) {
 		MaxLineSize: GetConfig().MaxRecordSize,
 		MustExist:   true,
 		Follow:      true,
-		Location:    &tail.SeekInfo{-limit, os.SEEK_END},
+		Location:    location,
 		ReOpen:      false,
 		Poll:        false,
 		RateLimiter: rateLimiter})
+
+	// IMPORTANT: this registration happens everytime app restarts
+	if shouldInitialize {
+		tracker.InitializeChildNode(instance.DockerId, filename, t.Location.Offset)
```

Comment:
We could use t.Tell() but we already have the current offset available to us, why do we need to use t.Tell()?